geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/index/RecordIdxStrategyTest.scala

```diff
@@ -98,30 +98,30 @@ class RecordIdxStrategyTest extends Specification with TestWithDataStore {
       val query = new Query(sftName, Filter.INCLUDE)
       query.getHints.put(SAMPLING, new java.lang.Float(.5f))
       val results = runQuery(query).toList
-      results must haveLength(10)
+      results.length must beLessThan(20)
```

Comment:
Did the test data change?  The counts have changed.

Reply:
all the sampling tests are dependent on the number of regions/splits/scans that are actually executed, as sampling is only done 'locally' (in a given scan range) and not globally. the old tests were exactly testing the behavior of the mock accumulo instance, so i relaxed it to just ensure that it was doing *some* sampling.